typescript/cli/src/tests/warp-apply.e2e-test.ts

```diff
@@ -0,0 +1,132 @@
+import { expect } from 'chai';
+import { Wallet } from 'ethers';
+import { beforeEach } from 'mocha';
```

Comment:
Is this import necessary? I thought the before/after/it fns just worked

Reply:
I would like each test to run with a freshly deployed warp route. One example is that the tests might burn the owner so subsequent updates may not work.

Reply:
RIght, but what I'm asking is, if you simply delete this line, does the test work the same?

Reply:
Ohhhhhhhhhh....

Yes they do :)